pkg/controllers/restore/controller.go

```diff
@@ -606,6 +606,17 @@ func (h *handler) restoreResource(restoreObjInfo objInfo, restoreObjData unstruc
 	name := restoreObjInfo.Name
 	namespace := restoreObjInfo.Namespace
 	gvr := restoreObjInfo.GVR
+	// TEMPORARY HOTFIX: Don't restore secrets of type fleet.cattle.io/cluster-registration-values
+	if gvr.Resource == "secrets" {
+		secretType, found, err := unstructured.NestedString(obj.Object, "type")
+		if err != nil {
+			return err
+		}
+		if found && secretType == "fleet.cattle.io/cluster-registration-values" {
+			logrus.Infof("Skip restoring the secret %s/%s since it has type fleet.cattle.io/cluster-registration-values", namespace, name)
```

Comment:
Suggesting to prefix message with `restoreResource: ` as done in all other log messages across this func.
Another nit is going with `Skipping` over `Skip` to align with format of other log messages (may be wrong on this one since it follows with `restoring`).

Reply:
I'll fix this